### PR TITLE
Privacy notice added to 'About' section of docs - fixes #85

### LINKED ISSUE #85: Privacy notice in redesigned docs
We now have a [privacy notice](http://beneficial-ownership-data-standard.readthedocs.io/en/master/privacy-notice.html) in the master branch of the docs from #84 .

We'll also need to have this in our re-designed documentation. Assigning to @kd-ods in the first instance to think about design aspects.

### PR DESCRIPTION
**Pull request by kd-ods**
In the course of looking at the privacy notice file, I realise that there is another problem. There's some [iframe code](https://github.com/openownership/data-standard/blob/0.2-dev-privacy-85/docs/about/privacy-notice.md) - a Matomo form? - which isn't working for some reason.

When I build the docs locally, the iframe is stripped out when I view the browser's HTML (in Chromium & Firefox).

I'm adding this PR anyway since the problem may not be in the privacy-notice.md file itself. I'll ask James for a quick review and work out whether to leave the Matomo iframe out in the file.

### CODE REVIEW

**Comment by kd-ods** [comment]
@odscjames will look into the iframe problem separately as part of non-sprint work (https://opendataservices.plan.io/issues/21398), so I think we can go ahead and merge this once @siwhitehouse has cast his eye over it.

File: docs/about/privacy-notice.md [hunk 1]
```diff
@@ -3,17 +3,15 @@ Privacy Notice
...
-Data controller: <br/>
-<br/>
-Contact us if would like a copy of the information held on you or if you believe that any information we are holding on you is incorrect or incomplete.
+You can contact us at OpenOwnership (as the data controller) if would like a copy of the information held on you or if you believe that any information we are holding on you is incorrect or incomplete.
 
 You have the following rights concerning this data:
 
-*   Right to be informed, which is the purpose of this privacy notice
+*   Right to be informed, which is the purpose of this privacy notice.
 *   Right to Access, Rectification, Erasure, and to Restrict Processing. Note that the right to Erasure and Restrict Processing are balanced against our legitimate interests. Where relevant, you need to provide information to re-identify yourself from our pseudonymised data, see [GDPR Article 11](https://gdpr-info.eu/art-11-gdpr/)
```
Comment (siwhitehouse) [inline docs/about/privacy-notice.md]: Should this include a full stop after `[GDPR Article 11](https://gdpr-info.eu/art-11-gdpr/)` for consistency?

**Comment by kd-ods** [comment]
Sorry, @siwhitehouse - I didn't realise that fixing that full-stop would prompt a re-review!

Once you've re-approved this PR, it can be merged and the branch deleted. (Of all the outstanding PRs it will be the easiest to knock off.) cc @ScatteredInk

**Comment by odscjames** [comment]
The conversion to RST removed the privacy iFrame. Commit https://github.com/openownership/data-standard/commit/174c39debafb423a02b160b238ae96170f987e45

This puts them back:

https://github.com/openownership/data-standard/pull/230

**Pull request merged**